cla_backend/apps/legalaid/migrations/0020_fill_missing_outcome_codes.py

```diff
@@ -0,0 +1,38 @@
+# coding=utf-8
+from __future__ import unicode_literals
+import logging
+
+from django.db import models, migrations
+
+logger = logging.getLogger(__name__)
+
+
+def re_denormalize_outcome_codes_to_cases(apps, schema_editor):
+    try:
+        from cla_eventlog.constants import LOG_LEVELS, LOG_TYPES
+        from cla_eventlog.models import Log
+    except ImportError:
+        pass
+    else:
+        outcomes_that_should_be_denormed = Log.objects.filter(level=LOG_LEVELS.HIGH, type=LOG_TYPES.OUTCOME)
+        outcomes_missing_denormed_code = outcomes_that_should_be_denormed.filter(case__outcome_code='')
```

Comment:
❓Is the filter implicitly using `ORDER BY created`, or in other words, is there any chance the time-ordering of `Log` entries matters?

Ie. if there were two or more HIGH outcome entries, would we overwrite a case outcome to an earlier entry?

Reply:
Yes, great spot. Explicitly reversed the queryset ordering to result in the latest HIGH+OUTCOME entry for a case getting written last.